src/assets/signals/buttons/pointer.svg

```diff
@@ -0,0 +1 @@
+<svg width="6" height="10" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.564 4.622a.5.5 0 0 1 0 .756L.827 9.48A.5.5 0 0 1 0 9.102V.898A.5.5 0 0 1 .827.52l4.737 4.102z" fill="#2F354D"/></svg>
```

Comment:
that should be in our storybook. Ask Anastasya about actual icons. She is working on it right now

Reply:
Yes, it is on storybook. But we can add background image to svg circle only as URL. In our storybook it is using as icon(not url for direct access). @haritonasty